CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ All notable changes to [bpmnlint-plugin-camunda-compat](https://github.com/camun
 
 ___Note:__ Yet to be released changes appear here._
 
+## 2.10.0
+
+* `FIX`: make `camunda-bpmn-moddle` a development dependency ([#142](https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/142))
+* `DEPS`: update dependencies ([#142](https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/142))
```

Comment:
Should this be a patch release as we don't add any new features and just bug-fixes? I am not 100% sure what's the correct way with semver

Reply:
I guess it could count as "substantial new functionality or improvements are introduced within the private code" and a minor version is fine 😄 

https://semver.org/#spec-item-7

Reply:
I usually do a minor bump in case larger changes happened. And this one clearly was a larger cleanup.

Reply:
Agreed! Just needed to freshen up on my semver reading first